src/sources/internal_logs.rs

```diff
@@ -99,17 +100,21 @@ mod tests {
 
     #[tokio::test]
     async fn receives_logs() {
+        let test_id: u8 = rand::random();
         let start = chrono::Utc::now();
         trace::init(false, false, "debug");
         trace::reset_early_buffer();
-        error!(message = "Before source started.");
+        error!(message = "Before source started.", %test_id);
 
         let rx = start_source().await;
 
-        error!(message = "After source started.");
+        error!(message = "After source started.", %test_id);
 
         sleep(Duration::from_millis(1)).await;
-        let events = collect_ready(rx).await;
+        let mut events = collect_ready(rx).await;
+        events.retain(|event| {
+            event.as_log().get("test_id") == Some(&Value::from(test_id.to_string()))
+        });
```

Comment:
This ends up repeatedly generating the `Value`. It's harmless since it's a test, but `rustfmt` may skip the extra block expansion this way:
```suggestion
        let test_id = Value::from(test_id.to_string());
        events.retain(|event| event.as_log().get("test_id") == Some(&test_id));
```